packages/components/src/CONTRIBUTING.md

```diff
@@ -0,0 +1,80 @@
+# Contributing components
+
+These guidelines document how to contribute to components in the [`@wordpress/components`](https://www.npmjs.com/package/@wordpress/components) npm package, or “WordPress Components”. This can be addition, modification, or deprecation.
+
+## Does it belong in the component library?
+
+A component library should include what’s shared across many products and omit what’s not. Questions to help determine if a component should be added:
+
+- Is it relevant to other products/plugins?
+- Is there overlap with any existing components?
+- How much will it cost to make and maintain?
```

Comment:
The word "cost" here might be conceptually difficult, since the first instinct to think of it in terms of money, rather than time or opportunity costs.